timescaledb/overview/release-notes/index.md

```diff
@@ -21,6 +21,20 @@ Want to stay up-to-date with new releases? Subscribe to get
 * Introduce fixed schedules for background jobs and the ability to check job errors.
 * Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.
 
+## 2.9.2 (2023-01-26)
```

Comment:
Why does the [`CHANGELOG.md` in the 2.9.x branch](https://github.com/timescale/timescaledb/pull/5210/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR12) has a different date than the release notes in this PR and the [canonical Changelog in the main project](https://github.com/timescale/timescaledb/blob/main/CHANGELOG.md#292-2023-01-26)?

Do we need to revisit the branching process for release branches to make sure that no changes are introduced or are you going to reverse rebase/merge afterwards?

Reply:
@iroussos 
Changes of #5172 is available in release PR (#5120), however its not mentioned in changelog. According to @svenklemm we don't specify PR's which fix only testfiles. Thus you see few missing PRs when compared against
https://github.com/timescale/timescaledb/milestone/66

Fixed the date. Thanks.

Reply:
Thank you @sb230132, you are right I forgot about that! Everything looks good to me, approving on my side :-)